ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2017.2.1f1
+m_EditorVersion: 2017.2.1f1
```

Comment:
Please revert this file.

Reply:
https://github.com/radicalad/MixedRealityToolkit-Unity/blob/MRDLSolver/ProjectSettings/ProjectVersion.txt - it looks fine to me?